packages/algoliasearch-helper/test/spec/RecommendParameters/methods.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
just thinking out loud, but are the tests allowed to be typescript here? could be interesting

Reply:
I've checked, ts files don't work run on algoliasearch-helper with its current configuration.